pkgs/development/libraries/audiofile/default.nix

```diff
@@ -27,7 +27,7 @@ stdenv.mkDerivation rec {
   };
 
   # fix build with gcc9
-  NIX_CFLAGS_LINK = lib.optional (stdenv.system == "i686-linux") "-lgcc";
+  NIX_CFLAGS_LINK = lib.optional (stdenv.system == "i686-linux" && !(stdenv.hostPlatform.useLLVM or false)) "-lgcc";
```

Comment:
I think you want to check `stdenv.cc.isClang` instead?

Reply:
I am not sure with Clang only; during my tests, I used a pure LLVM toolchain so I wouldn't be sure to check only for `clang`.

Reply:
```
nix-repl> llvmPackages.stdenv.cc.isClang
true
```